packages/vitepress-plugin-git-changelog/src/vite/index.ts

```diff
@@ -80,7 +80,7 @@ export function GitChangelog(options: {
         // get change log of each file
         // const raw = await git.raw(['diff-tree', '--no-commit-id', '--name-only', '-r', log.hash])
         const raw = await git.raw(['diff-tree', '--no-commit-id', '--name-status', '-r', '-M', log.hash])
-        delete log.body
+        // delete log.body
```

Comment:
Should we remove such commented code in this PR or add `// TODO:` for it to remove later?

Should we explain why we have such code left here?

Reply:
We can create a new issue to discuss moving more work to build timen.

Reply:
Created https://github.com/nolebase/integrations/issues/45